### PR TITLE
RAINCATCH-1186 API docs review 

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation
Review API documentation generated by typedoc 

## Description
fix any issues with in raincatcher-core so they will render in the api typedoc docs. Mainly fix relative links.

## Progress
- [x] Documentation

### CODE REVIEW

File: client/datasync-client/src/DataManager.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ import * as _ from 'lodash';
 /**
  * Wrapper class around sync to allow easier operations on a single data set.
  *
- * Sor more information on the sync client API,
- * See https://tinyurl.com/yc5q3rj3
+ * For more information on the sync client API,
+ * See https://github.com/feedhenry/fh-sync/blob/master/docs/sync_service.adoc
```
Comment (wtrocki) [inline client/datasync-client/src/DataManager.ts]: Agreement was to point to portal for the moment as documentation is outdated.

File: client/datasync-client/src/DataManager.ts [hunk 2]
```diff
@@ -6,8 +6,8 @@ import * as _ from 'lodash';
 /**
  * Wrapper class around sync to allow easier operations on a single data set.
  *
- * Sor more information on the sync client API,
- * See https://tinyurl.com/yc5q3rj3
+ * For more information on the sync client API,
```
Comment (wtrocki) [inline client/datasync-client/src/DataManager.ts]: , => .

**Comment by wtrocki** [comment]
Looks really good.
@austincunningham  Out of curiosity -  
Did you checked if there are some missing comments?

**Comment by wtrocki** [comment]
Took a quick glance into render myself. It looks good. I found some issues around interfaces - we no longer using them. Going to create follow up issues.

**Review by austincunningham — COMMENTED** [review]
Missing comments around tests mainly. Didn't think they were needed.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**